main.js

```diff
@@ -0,0 +1,30 @@
+// Get array data from api to variable
+function fetchData(cb, func) {
+  $.ajax({
+    url: "https://randomuser.me/api/?results=15",
+    success: function (response) {
```

Comment:
The parameter name can be the type of data you are receiving back, for example: `users`